src/hooks/usePoolTickData.ts

```diff
@@ -140,8 +142,17 @@ function useTicksFromSubgraph(
   currencyB: Currency | undefined,
   feeAmount: FeeAmount | undefined
 ) {
+  const { chainId } = useWeb3React()
   const poolAddress =
-    currencyA && currencyB && feeAmount ? Pool.getAddress(currencyA?.wrapped, currencyB?.wrapped, feeAmount) : undefined
+    currencyA && currencyB && feeAmount
+      ? Pool.getAddress(
+          currencyA?.wrapped,
+          currencyB?.wrapped,
+          feeAmount,
+          undefined,
+          chainId ? V3_CORE_FACTORY_ADDRESSES[chainId] : undefined
```

Comment:
Why is this necessary?
Also (as a double-check) is there anywhere else where this is necessary?

Reply:
Pool.getAddress() in the v3-sdk [originally used a constant factory address](https://github.com/Uniswap/v3-sdk/pull/124/files) (Celo is the first chain to use different addresses)

Reply:
What about the other instance of `Pool.getAddress`, on L61?

Reply:
Yeah solid point, it seems the call was still working even with the wrong pool address. I've added the change and tested ✅